### DISCUSSION TITLE
[Eclipse IDE] Default breakpoints

### DISCUSSION THREAD

**Discussion by wolfgang-ch** [mytourbook/mytourbook#977] [General]
These `Java Exception Breakpoints` can help to easier find out of bounds exceptions when they are always active

![image](https://user-images.githubusercontent.com/1283445/210236025-fd1c1699-2d24-45a6-a23b-e4dde6cb4358.png)

**Comment by FJBDev** [comment]
Does it mean it will stop the program execution (instead of continuing) when those exceptions are caught ?

**Reply by wolfgang-ch** [reply]
> Does it mean it will stop the program execution (instead of continuing) when those exceptions are caught ?

The program will stop and you can debug it

I also use the breakpoint import/export feature for these breakpoints to restore them easily when I accidentally delete them

![image](https://user-images.githubusercontent.com/1283445/211470404-43375767-e9f5-4863-bbc5-a87b5fbed630.png)

**Reply by wolfgang-ch** [reply]
This exception just happened in my IDE, `positionIndex_1` was `1` but the array had only 1 element. So I could easily see the reason why it happened and can fix it.

![image](https://user-images.githubusercontent.com/1283445/211484002-a1456dcb-1192-4d1b-992b-78da5768e7cc.png)
